KDagal-HTML-and-CSS-Week1/Blog Page.html

```diff
@@ -0,0 +1,73 @@
+        <img src="Honey.jfif" alt="Raw Honey Picture">
+    </header>
+
+    <main> 
+     <section>
+         <h2>Here are 3 of the Importance of raw honey</h2>
+         <article class="art1">
+             <h3>1. A good source of antioxidants</h3>
+              <p>
+                 Raw honey contains an array of plant chemicals that act as antioxidants. Some types of honey have as many antioxidants as fruits and vegetables.<br/>  
+                 Antioxidants help to protect your body from cell damage due to free radicals. Free radicals contribute to the aging process and may also contribute to the development of chronic diseases such as cancer and heart disease.<br/> 
+                 Research shows that antioxidant compounds in honey called polyphenols may play a role in preventing heart disease.
+                 <br/> <a href="https://www.healthline.com/health/food-nutrition/top-raw-honey-benefits#boost-immunity">Click here more Info</a>
+              </p>
+         </article>
+     
+     
+         <article class="art2">
+              <h3>2. Antibacterial and antifungal properties</h3>
+               <p>
+                  Research has shown that raw honey can kill unwanted bacteria and fungus. It naturally contains hydrogen peroxide, <br/> 
+                  an antiseptic. Its effectiveness as an antibacterial or antifungal varies depending on the honey, <br/> b
+                  ut it’s clearly more than a folk remedy for these kinds of infections.
+                  <br/><a href="https://www.healthline.com/health/food-nutrition/top-raw-honey-benefits#boost-immunity">Click here more Info</a>
+               </p>
+          </article>
+      
+
+      
+         <article class="art3">
+              <h3>3. Heal wounds</h3>
+                <p>
+                  Manuka honey is used in medical settings to treat wounds because it’s been found to be an effective germ killer and also aids in tissue regeneration.<br/> 
+                  Studies show that Manuka honey can boost healing time and reduce infection. <br/>
+                   Keep in mind that the honey used in hospital settings is medical grade, meaning it’s inspected and sterile. <br/> 
+                   It’s not a good idea to treat cuts with honey you buy from a store.
+                  <br/><a href="https://www.healthline.com/health/food-nutrition/top-raw-honey-benefits#boost-immunity">Click here more Info</a>
+             </p>
+         </article>
+        </section>
+    </main>
+
+    <aside>
+        <fieldset>
+          <legend><h3>Feedback Form</h3></legend>
+            <form>
+                <input type="text" name="Name" placeholder="Your full name" required> <br>
+                <input type="email" name="Email" placeholder="Enter your email address" required> <br>
+                <label>Do you believe raw honey is important for our health?</label> <input type="checkbox" name="checkbox" required> <br>
```

Comment:
With the checkbox being required, the answer to "Do you believe raw honey is important for our health?" is always going to be yes, or was that intended? 🍯